package.json

```diff
@@ -101,15 +101,8 @@
     "stylelint": "^13.13.1",
     "stylelint-config-recommended": "^5.0.0",
     "stylelint-config-styled-components": "^0.1.1",
-    "webpack": "^5.42.0",
-    "webpack-cli": "^4.7.2"
+    "webpack": "^5.51.1",
+    "webpack-cli": "^4.8.0"
   },
-  "dependencies": {
-    "@metamask/detect-provider": "^1.2.0",
-    "assert": "^2.0.0",
-    "https-browserify": "^1.0.0",
-    "os-browserify": "^0.3.0",
-    "stream-http": "^3.2.0",
-    "web3": "^1.3.5"
-  }
+  "packageManager": "yarn@3.0.1"
```

Comment:
Now that I removed this, I get the errors again, even though the dependency was updated @jacogr 
```
➜  apps git:(jlm-adapt-app-to-metamask-extension) ✗ yarn start
$ polkadot-dev-clean-build 
[webpack-cli] Failed to load '/Users/antoineestienne/GitHubRepo/apps/packages/apps/webpack.serve.cjs' config
[webpack-cli] Error: Cannot find module 'stream-http'
```